utils/mil_tools/nodes/label_me_bag.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+
+"""
+Converts bags to labelme images, or labelme annotations to bags.
+
+Uses a YAML file to specify what times and topics in a set of bag files
+will be used to produce images which are places in a corosponding
```

Comment:
Everything is actually spelled right here, they're just not the right words. I don't really know what that's called. Thanks for the tip, though.